biome.json

```diff
@@ -1,10 +1,7 @@
 {
   "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "extends": ["./node_modules/@cubing/dev-config/biome/biome.json"],
```

Comment:
Is `@cubing/dev-config` already in our package.json? Or are we pulling it in indirectly?

Reply:
It's already a dev dependency: https://github.com/cubing/icons/blob/93f675c2520a7dde550167016e8b8611906fc192/package.json#L34

We're already using it for `tsconfig.json`.